### PR TITLE
fix(libsass.html): correcting link to sass.js playground

### PR DESCRIPTION
**Pull request by rodneyrehm**
… just fixing the link target for "testing sass.js in the browser" :)

### CODE REVIEW

File: source/libsass.html.haml [hunk 1]
```diff
@@ -51,7 +51,7 @@ title: libSass
   %li#javascript
     :markdown
       ### JavaScript
-      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/sass.js/), too.
+      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/playground.sass.js/), too.
```
Comment (jina) [inline source/libsass.html.haml]: Hey @rodneyrehm  — do you mind fixing the houndci error in this and then I'll merge it in? Just make sure the code is around 78-ish characters.

Comment (rodneyrehm) [inline source/libsass.html.haml]: I'm not familiar with haml. will a simple indented line-break do the job?

Comment (jina) [inline source/libsass.html.haml]: Yep. It's actually in a markdown filter. Just a line break will do. Make sure it's at the same level of indention though. Thanks!

**Comment by rodneyrehm** [comment]
@jina there you go, 80 characters. Note that pretty much every other line in that file is *way* longer than that limit. Are you sure imposing limits like that on flowing text content is a good idea?

**Comment by jina** [comment]
It's been in the [Code Style Guide](http://sass-lang.com/styleguide/code) from the launch of the site, but no one has been paying attention to it.

The HoundCI checker will help make sure new pull requests going forward follow the guidelines.

It's for better code quality. :)

**Pull request merged**